app/src/main/java/org/opencord/aaa/impl/StateMachine.java

```diff
@@ -219,6 +259,24 @@ public void setSupplicantAddress(MacAddress supplicantAddress) {
         this.supplicantAddress = supplicantAddress;
     }
 
+    /**
+     * Sets the lastEapolRadiusPacketReceivedTime.
+     *
+     * @param lastEapolRadiusPacketReceivedTime timelastPacket was received
+     */
+    public  void setlastEapolRadiusPacketReceivedTime(long lastEapolRadiusPacketReceivedTime) {
+        this.lastEapolRadiusPacketReceivedTime = lastEapolRadiusPacketReceivedTime;
+    }
+
+    /**
+     * Gets the lastEapolRadiusPacketReceivedTime.
+     *
+     * @return lastEapolRadiusPacketReceivedTime
+     */
+    public long getlastEapolRadiusPacketReceivedTime() {
```

Comment:
As discussed, change the variable to lastPacketReceivedTime. Also, the name of getter and setter methods should have L CAPITAL i.e. Last instead of last (camel casing)

Reply:
Done